simulator/mips/mips_instr.cpp

```diff
@@ -342,14 +351,28 @@ static const Table<I> isaMapCOP0 =
     {0x04, { "mfc0",  mips_mfc0<I>, OUT_ARITHM, 0, 'N', Imm::NO, Src1::CP0_RT, Src2::ZERO, Dst::RD, MIPS_I_Instr} },
 };
 
+template<typename I>
+static const Table<I> isaMapCOP1 =
+{
+    {0x00, { "add.s", mips_add_s<I>, OUT_ARITHM,  0, 'S', Imm::NO, Src1::CP1_FS, Src2::CP1_FT, Dst::CP1_FD, MIPS_I_Instr} },
```

Comment:
I think we have to introduce a new class of instructions: OUT_FPU. These are no integer arithmetic nor integer multiplication.